Resources/Private/Templates/Backend/Library/Index.html

```diff
@@ -3,6 +3,99 @@
 
 <f:section name="content">
     <div class="neos-row-fluid">
-        <legend>TBD</legend>
+        <f:widget.paginate objects="{libraries}" as="paginatedLibraries"
+                           configuration="{itemsPerPage: 30, maximumNumberOfLinks: 7}">
+            <table class="neos-table">
+                <thead>
+                <tr>
+                    <th class="neos-span5">Library</th>
+                    <th class="neos-span1">Version</th>
+                    <th class="neos-span1">Contents</th>
+                    <th class="neos-span2">Contents using it</th>
+                    <th class="neos-span2">Libraries using it</th>
+                    <th class="neos-span1">&nbsp;</th>
+                </tr>
+                </thead>
+                <tbody>
+
+                <f:for each="{paginatedLibraries}" as="library" key="index">
+                    <tr>
+                        <td>
+                            <neos:link.module
+                                title="View Library"
+                                path="h5p/library"
+                                action="display"
+                                arguments="{library:library}">
+                                {library.title}
+                            </neos:link.module>
+                        </td>
+                        <td>
+                            {library.versionString}
+                        </td>
+                        <td>
+                            {library.contentDependencies -> f:count()}
+                        </td>
+                        <td>
+                            {library.contents -> f:count()}
+                        </td>
+                        <td>
+                            {library.librariesUsingThisLibrary -> f:count()}
+                        </td>
+                        <td class="neos-action">
+                            <div class="neos-pull-right">
+                                <f:if condition="{library.upgradeAvailable}">
```

Comment:
wordpress shows upgrade buttons for every runnable library, always inactive and greyed out until an upgrade is available. We could mirror this behavior by checking for library.runnable before showing the upgrade button. 
I decided agains this behavior of the UI and so we only show the upgrade button when there actually is an upgrade and thus we never check for library.runnable as a display condition.